docs/core/extensions/console-log-formatter.md

```diff
@@ -81,15 +81,15 @@ info: Microsoft.Hosting.Lifetime[0]
 
 By default, the `Simple` console log formatter is selected with default configuration. You change this by calling `AddJsonConsole` in the *Program.cs*:
 
-:::code language="csharp" source="snippets/logging/console-formatter-json/Program.cs" highlight="7-15":::
+:::code language="csharp" source="snippets/logging/console-formatter-json/Program.cs" highlight="5-13":::
 
 Alternatively, you can also configure this using logging configuration, such as that found in the _appsettings.json_ file:
 
 :::code language="json" source="snippets/logging/console-formatter-json/appsettings.json" highlight="14-23":::
 
 Run the app again, with the above change, the log message is now formatted as JSON:
 
-:::code language="json" source="snippets/logging/console-formatter-json/example-output.json":::
+:::code language="json" source="snippets/logging/console-formatter-json/example-output.txt":::
```

Comment:
`.txt` file extension seems out of sync with `language="json"` and the following alert box talking about the JSON console formatter

Reply:
This is intentional, it's to represent the example logging output as formatted by JSON. However, each time it logs, its entry is a JSON object, but it's not formatted holistically as an array. As such, it's invalid JSON (and Visual Studio now throws an error during compilation). I do, however, want it to be displayed in docs as JSON.